src/Features/VisualBasic/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.ReferenceRewriter.vb

```diff
@@ -62,6 +62,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.InlineTemporary
                 Return MyBase.VisitIdentifierName(node)
             End Function
 
+            Public Overrides Function VisitNameColonEquals(node As NameColonEqualsSyntax) As SyntaxNode
+                If node.IsParentKind(SyntaxKind.SimpleArgument) AndAlso
+                    node.Parent.IsParentKind(SyntaxKind.TupleExpression) Then
+
+                    Return node
```

Comment:
this needs documentation as to why you are explicitly not changing anything in this situatoin.